src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefReadonlyReturnTests.cs

```diff
@@ -450,15 +450,15 @@ class Program
 
             var comp = CreateCompilationWithMscorlib45(text, new[] { ValueTupleRef, SystemRuntimeFacadeRef });
             comp.VerifyDiagnostics(
-                // (6,25): error CS1031: Type expected
+                // (6,33): error CS0106: The modifier 'readonly' is not valid for this item
```

Comment:
Not sure what's going on here. I wouldn't expect 'ref' or 'readonly' to be parse as modifiers in this case, but as part of the type.